default_app/preload.ts

```diff
@@ -1,7 +1,9 @@
 import { ipcRenderer } from 'electron'
 
-function initialize () {
-  const electronPath = ipcRenderer.sendSync('bootstrap')
+const bootstrap = ipcRenderer.invoke('bootstrap')
```

Comment:
Why define it out here, seems like a strange pattern compared to just

```js
const electronPath = await ipcRenderer.invoke('bootstrap')
```

Reply:
the idea was that the promise may already be resolved by the time `DOMContentLoaded` is fired